code/game/objects/items/trader.dm

```diff
@@ -1220,6 +1219,143 @@ var/global/list/alcatraz_stuff = list(
 		return FALSE
 	return TRUE
 
+/obj/structure/closet/crate/internals/cloudnine
+	name = "Cloud IX engineering crate"
+	desc = "The Cloud IX engineering facility hangs in the atmosphere of the eponymous gas giant. But are the workers happy? Nein."
+
+//3+4+3=10
+var/global/list/cloudnine_stuff = list(
+	//3 of a kind
+	/obj/item/airshield_projector,/obj/item/airshield_projector,/obj/item/airshield_projector,
+	//2 of a kind
+	/obj/item/vaporizer,/obj/item/vaporizer,
+	/obj/item/device/multitool/omnitool,/obj/item/device/multitool/omnitool,
+	//1 of a kind
+	/obj/item/clothing/gloves/golden,
+	/obj/machinery/power/antiquesynth,
+	/obj/item/weapon/am_containment/decelerator
+	)
+
+/obj/structure/closet/crate/internals/cloudnine/New()
+	..()
+	for(var/i = 1 to 5)
+		if(!cloudnine_stuff.len)
+			return
+		var/path = pick_n_take(cloudnine_stuff)
+		new path(src)
+
+/obj/item/clothing/gloves/golden
+	name = "golden gloves"
+	desc = "An impressive fashion statement. Gold is an excellent conductor, meaning these won't help much against shocks. The tag on the inside dares: touch the supermatter."
```

Comment:
I would also change the gloves' description to "An impressive fashion statement. Gold is an excellent conductor, meaning these won't help much against shocks. There's something else intertwined with the gold, but you can't put recognize it. The tag on the inside dares: touch the supermatter." To imply that it isn't the gold, but rather the [REDACTED] whatever what makes the gloves special.